Week 1/18/Q2_Daebiangmi_18.c

```diff
@@ -0,0 +1,23 @@
+// Write a program to input a positive number and reverse the digits of the number.
+
+#include <stdio.h>
+#include <conio.h>
+
+void main()
+{
+  int n, rem, r = 0;
+ 
+  printf("\n Enter a number: \n");
+  scanf("%d", & n);
+

```

Comment:
First check if the number is positive or not then find the reverse if it is positive